### ISSUE TITLE
Training details

### ISSUE THREAD

**Issue by wonjininfo** [yg211/summary-reward-no-reference#2]
Hi,
Congratulations on your work and thanks for sharing your code! 

I am doing a research on summarization and now I am trying to use the code.
I would like to train the model on the other dataset.
Would you please share some details on the training steps when training the model on CNN-DM dataset? 
Such as hyperparameters, the time spent to train the code and when (=which epoch) did you stoped the training.

ps) 50 epochs of training, which is the example command-line option, took only a few minutes (like less than an hour) on my GPU. Is this normal?

Thanks,
Wonjin
